hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/compact/TestInlineCompaction.java

```diff
@@ -93,6 +96,50 @@ public void testSuccessfulCompactionBasedOnNumCommits() throws Exception {
     }
   }
 
+  @Test
+  public void testSuccessfulCompactionBasedOnNumAfterCompactionRequest() throws Exception {
+    // Given: make 4 commits
```

Comment:
it would be more readable, if you use `step 1, step 2` to describe your intention. WDYT?

Reply:
Sure already add them in comment lines